Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN \
       nodejs \
       postgresql-client \
       yarn \
-      pkg-config \
+      pkg-config mesa-utils xvfb libgl1-mesa-dri libglapi-mesa libosmesa6 pkg-config x11proto-xext-dev xserver-xorg-dev libxext-dev libxi-dev \
```

Comment:
Having `pkg-config` one time should be enough.

Also, if you want to separate those packages from the rest, maybe just add another `apt-get install,` then you can also use shell comments in between the commands.